python/GafferMLUI/__init__.py

```diff
@@ -38,5 +38,7 @@
 from . import InferenceUI
 from . import ImageToTensorUI
 from . import TensorToImageUI
+from . import InferenceUI
```

Comment:
Duplicate of line 38?

Reply:
addressed here a879886e6db603f237e49c79c8d18a456f0acf04